tests/saw/spec/handshake/s2n_handshake_io.cry

```diff
@@ -512,7 +512,7 @@ SERVER_CHANGE_CIPHER_SPEC = 13
 SERVER_FINISHED = 14
 ENCRYPTED_EXTENSIONS = 15
 SERVER_CERT_VERIFY = 16
-APPLICATION_DATA = 18
+APPLICATION_DATA = 19
```

Comment:
What is the reason for this change?

Reply:
I inserted the new END_OF_EARLY_DATA message type before APPLICATION_DATA in the message_type_t enum: https://github.com/awslabs/s2n/blob/612ae83ca84a40f414b7b8db5090caf11fc55adc/tls/s2n_handshake.h#L71-L74

That changed the value of APPLICATION_DATA.